src/zenml/cli/stack.py

```diff
@@ -1143,16 +1143,28 @@ def _import_stack_component(
 @stack.command("import", help="Import a stack from YAML.")
 @click.argument("stack_name", type=str, required=True)
 @click.argument("filename", type=str, required=False)
+@click.option(
+    "--ignore-version-mismatch",
+    is_flag=True,
+    help="Import stack components even if the installed version of ZenML "
+    "is different from the one specified in the stack YAML file",
+)
 @click.pass_context
 def import_stack(
-    ctx: click.Context, stack_name: str, filename: Optional[str]
+    ctx: click.Context,
+    stack_name: str,
+    filename: Optional[str],
+    ignore_version_mismatch: bool,
 ) -> None:
     """Import a stack from YAML.
 
     Args:
         ctx: The click context.
         stack_name: The name of the stack to import.
         filename: The filename to import the stack from.
+        ignore_version_mismatch: Import stack components even if
+            the installed version of ZenML is different from the
+            one specified in the stack YAML file.
```

Comment:
It feels strange that the yaml file generated has a zenml version at all or even a zenml version different for the zenml package that calls this command, is there no way of avoiding this?

Reply:
The generated YAML needs to have a version number because syntax for some stack components might differ across versions, if we don't have entirely new components altogether. As for why it differs from the installed zenml version: the version on the stack YAML file comes from the stack recipe (it shows that the given yaml is valid for this specific version at least) and is therefore independent of the installed zenml on the user end.